docs/en/query_language/functions/string_replace_functions.md

```diff
@@ -60,7 +60,7 @@ SELECT replaceRegexpAll('Hello, World!', '.', '\\0\\0') AS res
 
 ```
 ┌─res────────────────────────┐
-│ HHeelllloo,,  WWoorrlldd!! │
+│ HHeelllloo,, WWoorrlldd!! │
```

Comment:
you should have skipped this kind of ascii-tables, or maybe even all code blocks

Reply:
About all blocks not sure, cause could appear situation like this:
``` sql
SELECT Carrier, count(*) FROM ontime WHERE DepDelay>10**_ _**AND Year = 2007 GROUP BY Carrier ORDER BY count(*) DESC
```
Where '_' means space.
Should i ignore these spaces too?

Reply:
> Should i ignore these spaces too?
No, but to distinguish between these cases you'll probably have to review all of them manually.

Reply:
Ok